pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnMetadata.java

```diff
@@ -114,12 +109,6 @@ public boolean equals(Object o) {
         if (nameWithCase != null ? !nameWithCase.equals(that.nameWithCase) : that.nameWithCase != null) {
             return false;
         }
-        if (!Arrays.deepEquals(fieldNames, that.fieldNames)) {
-            return false;
-        }
-        if (!Arrays.deepEquals(positionIndices, that.positionIndices)) {
-            return false;
-        }
```

Comment:
Is it necessary to compare the `DecoderExtraInfo`?

Reply:
thanks for the remind, fixed as suggestion.